docs/release_notes/0.162.0.md

```diff
@@ -0,0 +1,14 @@
+# Release v0.161.0
```

Comment:
```suggestion
# Release v0.162.0
```

Reply:
The release drafter is not very reliable.